### DISCUSSION TITLE
Where do you create the MSCS folder?

### DISCUSSION THREAD

**Discussion by albercuba** [MinecraftServerControl/mscs#334] [Q&A] [answered]
Hello everyone,
I have some questions.
I have a server for my kid and his school friends.
I use Ubuntu and the Minecraft folder is under **/etc/opt/minecraft**

Where should I install MSCS?
Where should I run this code?

`git clone https://github.com/MinecraftServerControl/mscs.git && cd mscs`

Thanks

**Comment by sandain** [comment, marked as answer]
Hi @albercuba.

The easiest way to install MSCS is to use the provided Makefile by running the command `sudo make install`. This will take care of copying files to their correct locations and creating a needed system user.  See the [installation documentation](https://minecraftservercontrol.github.io/docs/mscs/installation) for a step-by-step guide. First, make sure you have all of the [dependencies](https://minecraftservercontrol.github.io/docs/mscs/installation#dependencies) installed, then take a look at the [Quick Start](https://minecraftservercontrol.github.io/docs/mscs/installation#quick-start) section.

Once installed,  take a look at the [Getting Started](https://minecraftservercontrol.github.io/docs/mscs/getting-started) section for how to create a new Minecraft world or import an existing one. If you are interested in running other sever types, like Forge, check out the section on [Adjusting World & Server Properties](https://minecraftservercontrol.github.io/docs/mscs/adjusting-world-server-properties).

If you have any issues, or have further questions, feel free to continue this thread.

**Reply by albercuba** [reply]
Thanks,
just wanted to be sure.
